.travis.yml

```diff
@@ -53,10 +53,10 @@ jobs:
             - sourceline: 'deb http://packages.cloud.google.com/apt cloud-sdk-trusty main'
               key_url: 'https://packages.cloud.google.com/apt/doc/apt-key.gpg'
           packages:
-            - doxygen
+            - cmake
```

Comment:
Why wasn't cmake installed before?

Reply:
Because we didn't compile doxygen from scratch (takes ~1min). That's not ideal, of course. Alternatively, we could put a static binary somewhere and download it.

Reply:
Ah yes - sorry I was thinking we needed `cmake` to build CBMC, but on most platforms we use `make`